packages/ckeditor5-image/src/image/imageblockediting.ts

```diff
@@ -115,10 +115,12 @@ export default class ImageBlockEditing extends Plugin {
 		conversion.for( 'upcast' )
 			.elementToElement( {
 				view: getImgViewElementMatcher( editor, 'imageBlock' ),
-				model: ( viewImage, { writer } ) => writer.createElement(
-					'imageBlock',
-					viewImage.hasAttribute( 'src' ) ? { src: viewImage.getAttribute( 'src' ) } : undefined
-				)
+				model: ( viewImage, { writer } ) => {
+					return writer.createElement(
+						'imageBlock',
+						viewImage.hasAttribute( 'src' ) ? { src: viewImage.getAttribute( 'src' ) } : undefined
+					);
+				}
```

Comment:
It seems like nothing changed here (or I am blind). It could be that you changed the notation to debug something and then did not revert it.